src/hooks/useDailyStatistics/useDailyStatistics.tsx

```diff
@@ -16,6 +19,9 @@ export type StatisticsHook = {
   }[];
   error?: Error;
   loading: boolean;
+  c13ntForUnit: (
+    unit: CampaignPolicy["quantity"]["unit"]
+  ) => CampaignPolicy["quantity"]["unit"];
```

Comment:
Why do we need to export this function for this particular hook?

Reply:
for the testing. destructure c13ntForUnit and mock hocks.

Reply:
For the testing, we don't need the c13ntForUnit to be actually instantiated, we can just mock it there. Notice that the utils is calling the main hook, which is weird because the test of utils do not need to have a direct dependency on `useDailyStatistics`

Reply:
have removed the dependency and using mocks.

Reply:
Can we delete this export then?

Reply:
Yes, deleted.